db/migrate/20171016124659_create_users.rb

```diff
@@ -0,0 +1,12 @@
+class CreateUsers < ActiveRecord::Migration[5.1]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :number
```

Comment:
number or mobile_number or phone_number?

Reply:
Changed this to phone_number